dags/landsat_scenes_sync/landsat_scenes_sync.py

```diff
@@ -31,18 +31,18 @@
     "retries": 0,
     "retry_delay": timedelta(minutes=15),
     "depends_on_past": False,
-    "start_date": datetime(2020, 4, 8),
-    "catchup": True,
-    "version": "0.6.2",
+    "start_date": datetime(2021, 1, 1),
+    "version": "0.7.0",
 }
 # [END default_args]
 
 # [START instantiate_dag]
 dag = DAG(
-    "landsat_scenes_sync",
+    "landsat_scenes_identifying",
```

Comment:
if u are changing this name, shouldnt the file name change too?/

Reply:
I reckon you've got a point, I'll change

Reply:
Done